lib/App/Sqitch/Engine/mysql.pm

```diff
@@ -313,24 +313,35 @@ sub begin_work {
     return $self;
 }
 
-# Override to try to acquire a lock on the string "sqitch working" without
-# waiting.
+# We include the database name in the lock name because that's probably the most
+# stringent lock the user expects. Locking the whole server with a static string
+# prevents parallel deploys to other databases. Yes, locking just the target
+# allows parallel deploys to conflict with one another if they make changes to
+# other databases, but is not a great practice and likely an exceptional
```

Comment:
"this is not"

Reply:
It follows "locking just the target", so "locking just the target is not" is grammatically correct.

Reply:
Right. But I think in that case you don't want a comment before the "but".